rpi23-gen-image.sh

```diff
@@ -350,7 +350,7 @@ if [ -r "/dev/mapping/${CRYPTFS_MAPPING}" ] ; then
   exit 1
 fi
 
-# To delete an old build
+# Don't clobber an old build folder
```

Comment:
- [ ] changing the comment to be inconsistent with the code inside the conditional ... is wrong.

Reply:
Should I revert back to the original line which was inconsistent with the code?

Reply:
No, it wasn't inconsistent upstream, you made it inconsistent with your change to the commands inside the conditional block.